cloudfoundry-operations/src/main/lombok/org/cloudfoundry/operations/domains/Domain.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2013-2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.cloudfoundry.operations.domains;
+
+import lombok.Builder;
+import lombok.Data;
+
+/**
+ * Domain object representation
+ */
+@Data
+public final class Domain {
```

Comment:
@LokeshN Based on the output for the CLI

```
$: cf domains
Getting domains in org bhale as bhale@pivotal.io...
name        status   
cfapps.io   shared   
```

we'd expect two fields `name` and `status`, plus a `domainId` to enable easy transfer back to the `-client` API.